libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/barChart/PluggableBarChart.tsx

```diff
@@ -148,24 +149,24 @@ export class PluggableBarChart extends PluggableColumnBarCharts {
         }
 
         return {
-            currentSort: [],
+            defaultSort: [],
             availableSorts: [],
         };
     }
     public getSortConfig(referencePoint: IReferencePoint): Promise<ISortConfig> {
-        const { buckets } = referencePoint;
+        const { buckets, properties } = referencePoint;
         const viewBy = getBucketItems(buckets, BucketNames.VIEW);
         const measures = getBucketItems(buckets, BucketNames.MEASURES);
         const disabled = viewBy.length < 1 || measures.length < 1;
-        const { currentSort } = this.getDefaultAndAvailableSort(
+        const { defaultSort, availableSorts } = this.getDefaultAndAvailableSort(
             referencePoint,
-            canSortStackTotalValue(referencePoint.buckets, referencePoint.properties),
+            canSortStackTotalValue(buckets, properties),
         );
         return Promise.resolve({
             supported: true,
             disabled,
-            currentSort,
-            availableSorts: [],
+            currentSort: defaultSort,
```

Comment:
Shouldn't this be `defaultSort` as well then?

Reply:
Not sure, because we use it in AD as current sort and once currentSort from reference point will be reused (with some validation) with every loop AD<->PV then currentSort will be better name imho